package.json

```diff
@@ -46,11 +49,13 @@
     "eslint": "^8.23.1",
     "eslint-config-twilio-react": "^2.0.0",
     "html-webpack-plugin": "^5.0.0",
+    "jest": "^29.3.1",
     "process": "^0.11.10",
     "raw-loader": "^4.0.2",
+    "ts-jest": "^29.0.5",
     "ts-loader": "^8.0.16",
     "typescript": "^4.1.3",
-    "webpack": "^5.21.2",
+    "webpack": "^5.75.0",
```

Comment:
Not sure if this is necessary. I changed it to check if webpack related errors will go away, but eventually realized it is because I did not set the webpack configurations properly

Reply:
It's fine to bump it, better to keep up to date